src/lib/writers/ConfigWriter.ts

```diff
@@ -20,10 +20,13 @@ export class ConfigFileWriter implements ConfigWriter {
 		this.display = display;
 	}
 
-	public async write(model: ConfigModel): Promise<void> {
+	public async write(model: ConfigModel): Promise<boolean> {
```

Comment:
Since this method isn't guaranteed to actually write anything, it now returns a boolean to indicate whether it did.